src/hooks/useGallery/useGallery.tsx

```diff
@@ -2,6 +2,13 @@ import * as React from 'react';
 
 import {GalleryContext} from './GalleryContext';
 
-export function useGallery() {
-    return React.useContext(GalleryContext);
+export function useGallery(onCloseCallback?: () => void) {
```

Comment:
Let's use an object as the args here, it will be useful if we want to add something else